### PR TITLE
Fixes the case when a fee estimation request is made before the FullNode has initialized a peak

### PR DESCRIPTION
**Pull request by aqk**
Fixes the case when a fee estimation request is made before the FullNode has initialized a peak

### CODE REVIEW

File: tests/fee_estimation/test_fee_estimation_rpc.py [hunk 1]
```diff
@@ -44,6 +44,26 @@ async def setup_node_and_rpc(
     return client, full_node_rpc_api
 
 
+@pytest_asyncio.fixture(scope="function")
+async def setup_node_and_rpc_no_blocks_farmed(
+    two_wallet_nodes_services: Tuple[List[Service[FullNode]], List[Service[WalletNode]], BlockTools],
```
Comment (altendky) [inline tests/fee_estimation/test_fee_estimation_rpc.py:49]: We could avoid setting up two wallets here I think?

File: chia/rpc/full_node_rpc_api.py [hunk 2]
```diff
@@ -789,13 +789,15 @@ async def get_fee_estimate(self, request: Dict) -> Dict[str, Any]:
...
         peak_with_timestamp = peak_height
-        while last_peak_timestamp is None:
-            peak_with_timestamp -= 1
-            block_record = self.service.blockchain.height_to_block_record(peak_with_timestamp)
-            last_peak_timestamp = block_record.timestamp
+        if peak:
+            while last_peak_timestamp is None:
+                peak_with_timestamp -= 1
+                block_record = self.service.blockchain.height_to_block_record(peak_with_timestamp)
+                last_peak_timestamp = block_record.timestamp
```
Comment (altendky) [inline chia/rpc/full_node_rpc_api.py:800]: How would you feel about this?  Seems a bit more clear as to the handling of the two cases.

```suggestion
        if peak is None:
            peak_height = uint32(0)
            last_peak_timestamp = uint64(0)
        else:
            peak_height = peak.height
            last_peak_timestamp = peak.timestamp
            peak_with_timestamp = peak_height
            while last_peak_timestamp is None:
                peak_with_timestamp -= 1
                block_record = self.service.blockchain.height_to_block_record(peak_with_timestamp)
                last_peak_timestamp = block_record.timestamp
```

Comment (aqk) [reply chia/rpc/full_node_rpc_api.py:800]: Yes, looks clearer to me.

File: tests/fee_estimation/test_fee_estimation_rpc.py (hunk 1 above)
Comment (aqk) [reply tests/fee_estimation/test_fee_estimation_rpc.py]: Ya, agreed

File: chia/rpc/full_node_rpc_api.py (hunk 2 above)
Comment (aqk) [reply chia/rpc/full_node_rpc_api.py]: @altendky Something is preventing me from using the github `Commit suggestion` button - I just pushed these changes up from local.

**Review by arvidn — APPROVED** [review]
I take it those tests failed before the fix, right?

File: chia/rpc/full_node_rpc_api.py (hunk 2 above)
Comment (altendky) [reply chia/rpc/full_node_rpc_api.py]: Yeah, I wasn't sure if it would work, sorry.  I cheated since it wouldn't let me create it over a deleted line.

**Review by altendky — APPROVED** [review]

**Pull request merged**